client/js/main.js

```diff
@@ -5,24 +5,16 @@ import store from './store/store';
 import router from './router';
 Vue.config.productionTip = false;
 
-async function initApp() {
-  // Make sure we try and reestablish a player's session if one exists
-  // We do this before the app mounts to preempt the reauth/logout logic
-  await store.dispatch('requestStatus');
+const app = new Vue({
+  vuetify,
+  store,
+  router,
+  render: (h) => h(App),
+}).$mount('#app');
 
-  const app = new Vue({
-    vuetify,
-    store,
-    router,
-    render: (h) => h(App),
-  }).$mount('#app');
-
-  // Expose app for testing
-  if (process.env.NODE_ENV === 'development' || window.Cypress) {
-    window.cuttle = {
-      app,
-    };
-  }
+// Expose app for testing
+if (process.env.NODE_ENV === 'development' || window.Cypress) {
+  window.cuttle = {
+    app,
+  };
```

Comment:
By moving the initial auth logic to the vue router implementation, we can simplify this file again.